packages/core/src/Cardano/types/Transaction.ts

```diff
@@ -100,3 +100,8 @@ export interface TxAlonzo extends NewTxAlonzo<TxBodyAlonzo> {
   body: TxBodyAlonzo;
   txSize: number;
 }
+
+export type TxInternals = {
```

Comment:
I think this name isn't great now that it's no longer in scope of the wallet. Let's try to come up with a better one

Reply:
`TxBodyWithHash`?

Reply:
Yeah that's a decent one, definitely better than TxInternals

Reply:
[b946473](https://github.com/input-output-hk/cardano-js-sdk/pull/429/commits/b94647391a315ba376c91cf345f44119410ea8c1)